### PR TITLE
Update headers

### PR DESCRIPTION
**Pull request by SandroMachado**
This PR updates the headers used by the SDK for the new headers used by the API.

### CODE REVIEW

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java [hunk 1]
```diff
@@ -302,7 +302,7 @@ public void commitWithMessageShouldReturnTheTransaction() throws Exception {
         Transaction transaction = adapter.getResult();
 
         for (Header header : request.getHeaders()) {
-            if (header.getName().compareToIgnoreCase("X-Bitreserve-OTP") == 0) {
+            if (header.getName().compareToIgnoreCase("OTP-Token") == 0) {
```
Comment (d-moreira) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java]: WDYT of changing the condition to `"OTP-Token".equalsIgnoreCase(header.getName())`? It's safer.

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java [hunk 2]
```diff
@@ -345,7 +345,7 @@ public void commitWithMessageAndOTPShouldReturnTheTransaction() throws Exception
...
```
Comment (d-moreira) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java]: ^^

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java [hunk 3]
```diff
@@ -388,7 +388,7 @@ public void commitWithMessageAndSecurityCodeShouldReturnTheTransaction() throws
...
```
Comment (d-moreira) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java]: ^^

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java [hunk 4]
```diff
@@ -431,7 +431,7 @@ public void commitWithOTPShouldReturnTheTransaction() throws Exception {
...
```
Comment (d-moreira) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java]: ^^

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java [hunk 5]
```diff
@@ -474,7 +474,7 @@ public void commitShouldReturnTheTransaction() throws Exception {
...
```
Comment (d-moreira) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java]: ^^

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/service/UserCardServiceTest.java [hunk 6]
```diff
@@ -84,7 +84,7 @@ public void confirmTransactionShouldReturnTheRequest() throws Exception {
         Request request = adapter.getRequest();
 
         for (Header header : request.getHeaders()) {
-            if (header.getName().compareToIgnoreCase("X-Bitreserve-OTP") == 0) {
+            if (header.getName().compareToIgnoreCase("OTP-Token") == 0) {
```
Comment (d-moreira) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/service/UserCardServiceTest.java]: ^^

**Comment by SandroMachado** [comment]
@d-moreira updated.

**Pull request merged**